src/node_modules/graph-view/test/test-node-ui.js

```diff
@@ -0,0 +1,31 @@
+var Node = require('graph-view/node-ui')
+var test = require('tape')
+var React = require('react')
+var r = require('r-dom')
+// var ReactTestUtils = require('react-test-utils')
+require('react-svg-patch')
+require('./testdom')('<html><body></body></html>')
```

Comment:
i'd rather we weren't using jsdom for this, we should be putting our browser-side tests in a module's `test-client` dir so they are run by `npm run test-client`, or directly with `./node_modules/.bin/run-browser src/node_modules/graph-view/test-client -b`, and we can setup our dom with normal dom operations (nice utility modules can be found in the [`npm-dom` namespace](https://github.com/npm-dom)).